examples/acme/docker-compose.yml

```diff
@@ -29,49 +27,52 @@ services :
         - bhsm
         - bmysql
         - brabbitmq
-      volumes:
-        - "./rate-limit-policies.yml:/go/src/github.com/letsencrypt/boulder/test/rate-limit-policies.yml:ro"
+      networks:
+        - default
 
   bhsm:
       image: letsencrypt/boulder-tools:2016-11-02
       hostname: boulder-hsm
-      networks:
-          default:
-              aliases:
-                - boulder-hsm
       environment:
           PKCS11_DAEMON_SOCKET: tcp://0.0.0.0:5657
       command: /usr/local/bin/pkcs11-daemon /usr/lib/softhsm/libsofthsm.so
       expose:
         - 5657
+      networks:
```

Comment:
Global comment for this file. Indentation is not the same everywhere. Could you please fix that ?